docs/src/model_providers_and_supported_functions.md

```diff
@@ -0,0 +1,30 @@
+```@meta
+CurrentModule = PromptingTools
+```
+
+# Model Providers and Supported Functions
+
+PromptingTools.jl routes AI calls through the use of subtypes of AbstractPromptSchema, which determine how data is formatted and where it is sent. (For example, OpenAI models have the corresponding subtype AbstractOpenAISchema, having the corresponding schemas - OpenAISchema, CustomOpenAISchema, etc.) This ensures that the data is correctly formatted for the specific AI model provider. 
+
+Below is an overview of the model providers supported by PromptingTools.jl, along with the corresponding schema information.
+
+| Abstract Schema         | Schema                    | Model Provider                         | aigenerate | aiembed | aiclassify | aiextract | aiscan | aiimage |
```

Comment:
I'm not sure how to add it, but for the aiscan/aiimage functions, I think it would be good to mention that they rely on the specific endpoints being implemented by the provider.

Reply:
Would a footnote below the table work? Or do you think there will be too many of them eventually?

Reply:
I don't feel strongly. 

I think it would work well for now - we don't have that many footnotes yet and we can always change it later!